client/controller/index.web.js

```diff
@@ -98,6 +98,25 @@ export function clientRouter( route, ...middlewares ) {
 	page( route, ...middlewares, smartHydrate );
 }
 
+function isEditorPath( path ) {
+	const splitPath = path.split( '/' ).filter( ( n ) => n );
+	switch ( splitPath[ 0 ] ) {
+		case 'page':
+		case 'post':
+			return true;
+		case 'edit':
+			if (
+				splitPath[ 1 ] &&
+				[ 'jetpack-testimonial', 'jetpack-portfolio' ].indexOf( splitPath[ 1 ] ) !== -1
```

Comment:
Using `Array.prototype.includes()` can make this a bit easier to read:

```suggestion
				[ 'jetpack-testimonial', 'jetpack-portfolio' ].includes( splitPath[ 1 ] )
```